Doc/whatsnew/3.14.rst

```diff
@@ -142,6 +142,11 @@ sqlite3
 * Remove :data:`!version` and :data:`!version_info` from :mod:`sqlite3`.
   (Contributed by Hugo van Kemenade in :gh:`118924`.)
 
+* Disallow using a sequence of params with named placeholders.
+  This had previously raised a :exc:`DeprecationWarning` since Python 3.12;
+  it will now raise a :exc:`sqlite3.ProgrammingError`.
+  (Contributed by Erlend E. Aasland in :gh:`118928` and :gh:`101693`.)
```

Comment:
We don't usually include the original issue where it was first deprecated, but it's not a bad idea.